src/simulations/set_generator.py

```diff
@@ -25,6 +25,7 @@
 from absl import logging
 import numpy as np
 from wfa_cardinality_estimation_evaluation_framework.common.analysis import relative_error
+import collections
```

Comment:
Are we using Google's style guide?  If so, standard library imports should come before others.

Reply:
Resolved in c5bdec7